docker/docker-bake.hcl

```diff
@@ -55,8 +55,9 @@ target "base-gpu" {
   dockerfile = "Dockerfile.base"
   tags = ["${IMAGE_NAME}:base-gpu-${IMAGE_TAG_SUFFIX}"]
   args = {
-    build_image = "pytorch/pytorch:1.12.1-cuda11.3-cudnn8-runtime"
-    base_immage = "pytorch/pytorch:1.12.1-cuda11.3-cudnn8-runtime"
+    python = "python3.9"
+    build_image = "nvidia/cuda:11.2.2-cudnn8-runtime-ubuntu20.04"
+    base_immage = "nvidia/cuda:11.2.2-cudnn8-runtime-ubuntu20.04"
```

Comment:
again just curious... any specific reason for downgrading from 11.3 to 11.2.2 or maybe they are same?

Reply:
Good point I'll check which Cuda platform can technically cover the most available GPUs. I picked this runtime as HF transformers uses it.